app/gateways/http/authentication/create_token.go

```diff
@@ -0,0 +1,90 @@
+package authentication
+
+import (
+	"encoding/json"
+	auth "github.com/patriciapedrosaa/transfer-me/app/domain/authentication"
+	http_server "github.com/patriciapedrosaa/transfer-me/app/gateways/http"
+	"net/http"
+)
+
+var (
+	ErrInvalidPayload = "invalid request payload"
+	ErrRequiredFields = "invalid fields"
+	ErrNotFound       = "not found"
+	ErrUnexpected     = "something went wrong"
+)
+
+type LoginRequest struct {
+	CPF    string `json:"cpf" validate:"required"`
+	Secret string `json:"secret" validate:"required"`
+}
+
+type CreateTokenResponse struct {
+	Token string `json:"Token" validate:"required"`
+}
+
+type LoginResponse struct {
+	Token string
+}
+
+func (h Handler) CreateToken(w http.ResponseWriter, r *http.Request) {
+	var body LoginRequest
```

Comment:
the content of those structs is well written, but what do you think to keep as the first one (LoginRequest) with naming bound by the route, like LoginRequest and LoginResponse for structs and Login for the handler method.

Besides, currently in your production code, you're using just CreateTokenResponse, although LoginResponse is being used in your tests. Since they're equal, wouldn't be better to keep just one of them?

Reply:
I agree! the duplicate code went unnoticed.
I've already made the adjustments and changed the method name too.